linkerd/docs/protocol-http.md

```diff
@@ -485,31 +485,6 @@ Key | Default Value | Description
 mixerHost | `istio-mixer` | Hostname of the Istio Mixer server.
 mixerPort | `9091` | Port of the Mixer server.
 
-<a name="http-engines"></a>
-## HTTP Engines
```

Comment:
do you think this will necessitate a 1.3 release (since it's breaking configs?)

Reply:
Yeah